modules/veteran/lib/tasks/veteran_tasks.rake

```diff
@@ -16,13 +16,15 @@ namespace :veteran do
     Veteran::Service::Organization.create(poa: '095', name: '095 - ITALIAN AMERICAN WAR VETERANS OF THE US, INC.')
     Veteran::Service::Organization.create(poa: '1NY', name: '1NY - SAMANTHA Y WARSHAUER')
     Veteran::Service::Representative.create(
+      representative_id: '98765',
       poa_codes: %w[A1Q 095 074 083 1NY],
       first_name: 'Tamara',
       last_name: 'Ellis',
       email: 'va.api.user+idme.001@gmail.com'
     )
     Veteran::Service::Representative.create(
-      poa_codes: %w[A1Q 095 074 083 1NY],
+      representative_id: '12345',
```

Comment:
FWIW, when @SlakrHakr and I were working on the cleaning up the existing POA data so that it would conform to the new constraints, we set John Doe's `representative_id` to `9999999999999`. (one more `9` than Tamara's id)